### PR TITLE
Added v8 version upgrade flow

### PR DESCRIPTION
**Pull request by MeirShpilraien**
The new flow is manually dispatch and will create a PR with the needed changes to update the default version.

### CODE REVIEW

File: .github/workflows/update_version.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        description: 'The version to update to'
+        type: string
+        required: true
+
+jobs:
+
+  update-v8-version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```
Comment (chayim) [inline .github/workflows/update_version.yml:16]: checkout@v4

File: .github/workflows/update_version.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      run: |
+        export V8_VERSION_CURRENT=`cat ./V8_DEFAULT_VERSION.txt`
+        echo "V8_VERSION_CURRENT=$V8_VERSION_CURRENT" >> "$GITHUB_OUTPUT"
+    - name: Check new version is geater than current version
+      env:
+        V8_VERSION_CURRENT: ${{ steps.get-current-v8-version.outputs.V8_VERSION_CURRENT }}
+      if: ${{ env.V8_VERSION_CURRENT >= inputs.v8_version }}
+      run: |
+        echo new version ${{ inputs.v8_version }} is not greater then current version ${{ env.V8_VERSION_CURRENT }}
+        exit 1
```
Comment (chayim) [inline .github/workflows/update_version.yml:28]: I think you can just do:
```if: <your stuff>
fail()
```

File: .github/workflows/update_version.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+  update-v8-version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Getting V8 version
+      id: get-current-v8-version
+      run: |
+        export V8_VERSION_CURRENT=`cat ./V8_DEFAULT_VERSION.txt`
+        echo "V8_VERSION_CURRENT=$V8_VERSION_CURRENT" >> "$GITHUB_OUTPUT"
```
[the remark this answers is not in the document]
Comment (MeirShpilraien) [reply .github/workflows/update_version.yml:21]: But its not a github variable, its the local variable defined on the line above..

File: .github/workflows/update_version.yml (hunk 2 above)
Comment (MeirShpilraien) [reply .github/workflows/update_version.yml:28]: How can I verified?

File: .github/workflows/update_version.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (chayim) [reply .github/workflows/update_version.yml:21]: I know, just safer with the "{", I'm a fan.

File: .github/workflows/update_version.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    - uses: actions/checkout@v4
+    - name: Getting V8 version
+      id: get-current-v8-version
+      run: |
+        export V8_VERSION_CURRENT=`cat ./V8_DEFAULT_VERSION.txt`
+        echo "V8_VERSION_CURRENT=${V8_VERSION_CURRENT}" >> "$GITHUB_OUTPUT"
+    - name: Check new version is geater than current version
+      env:
+        V8_VERSION_CURRENT: ${{ steps.get-current-v8-version.outputs.V8_VERSION_CURRENT }}
+      if: ${{ env.V8_VERSION_CURRENT >= inputs.v8_version }}
```
[the remark this answers is not in the document]
Comment (MeirShpilraien) [reply .github/workflows/update_version.yml:25]: Yeah good point, let me see how I fix it.

File: build.rs [hunk 5]
```diff
@@ -24,7 +24,7 @@ lazy_static::lazy_static! {
 
     static ref PROFILE: String = env::var("PROFILE").expect("PROFILE env var was not given");
 
-    static ref V8_DEFAULT_VERSION: &'static str = "11.8.172.16";
+    static ref V8_DEFAULT_VERSION: &'static str = include_str!("V8_DEFAULT_VERSION.txt").trim();
```
[the remark this answers is not in the document]
Comment (MeirShpilraien) [reply build.rs:27]: https://github.com/RedisGears/v8-rs/issues/70

File: .github/workflows/update_version.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (MeirShpilraien) [reply .github/workflows/update_version.yml]: @vityafx converted the version to a number, I believe it is good enough.

**Pull request merged**